pyproject.toml

```
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools", "setuptools_scm"]
```

```suggestion
requires = ["setuptools", "setuptools_scm", "wheel"]
```
Seems wheel will be needed too

---

@esc this needs addressing?

---

unlikely, `build` will build wheels fine without it, so I don't think it is strictly necessary

---

ok, thanks for taking a look!